.github/workflows/publish.yaml

```diff
@@ -0,0 +1,40 @@
+name: Publish
+
+on: workflow_dispatch
```

Comment:
This makes this workflow available for manual triggering on the project's Action tab.

Reply:
good call, we do not release that often so making it a conscious choice seems the right thing to do